service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderController.java

```diff
@@ -178,8 +179,13 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
         caseDetails.getData().put("orderCollection", orders);
 
         if (featureToggleService.isCloseCaseEnabled() && caseData.getOrderTypeAndDocument().getSubtype() != INTERIM) {
+            String orderAppliesToAllChildren = caseData.getOrderAppliesToAllChildren();
+            if (caseData.getChildren1().size() <= 1) {
+                orderAppliesToAllChildren = "Yes";
+            }
```

Comment:
would not it be better to have this logic inside caseData.getOrderAppliesToAllChildren()?

Reply:
sure, that is a good idea